packages/sfml.rb

```diff
@@ -0,0 +1,36 @@
+require 'buildsystems/cmake'
+
+class Sfml < CMake
+  description 'SFML is a simple, fast, cross-platform and object-oriented multimedia API.'
+  homepage 'https://www.sfml-dev.org/'
+  version '3.0.0'
+  license 'zlib'
+  compatibility 'aarch64 armv7l x86_64'
+  source_url 'https://github.com/SFML/SFML.git'
+  git_hashtag version
+  binary_compression 'tar.zst'
+
+  binary_sha256({
+    aarch64: '172e599bdf688d88a8e127c0c420699e560a8f728af4a422cf1452458f172103',
+     armv7l: '172e599bdf688d88a8e127c0c420699e560a8f728af4a422cf1452458f172103',
+     x86_64: '44aaaab7168a8c1cf62d317f1a9760e6e0a2d76edf0e9be337b178b239b421fa'
+  })
+
+  depends_on 'flac'
+  depends_on 'freetype'
+  depends_on 'libglvnd'
+  depends_on 'libvorbis'
+  depends_on 'libx11'
+  depends_on 'libxcursor'
+  depends_on 'libxi'
+  depends_on 'libxrandr'
+  depends_on 'eudev' # R
+  depends_on 'gcc_lib' # R
+  depends_on 'glibc' # R
+  depends_on 'libogg' # R
+  depends_on 'libxext' # R
+  depends_on 'libxfixes' # R
+  depends_on 'libxrender' # R
```

Comment:
Sort dependencies and add `# R` to all?

Reply:
The `# R` dependencies are the ones added by getrealdeps by checking the program at runtime-- the others I manually confirmed, so its good to differentiate the two.

Reply:
Does gettealdeps show the others as runtime deps too?

It only adds deps if you don't have them in there already.

Reply:
I'm not sure, admittedly. I know these ones are dependencies because the build failed until they were added.

Reply:
As it stands # R serves solely to notate that the reps were added by getrealdeps.rb

Reply:
Is this correct @satmandu?

Reply:
Yes but if you run `../tools/getrealdeps.rb <package name>` it will tell you what ALL the runtime deps are.

Reply:
Maybe this can be integrated into our tests to make sure we get it right?  Also, it would be nice to add a check for duplicate dependencies.

Reply:
This package is identical to the last 40 or so new packages that you or I have added recently, in that we add the dependencies we know about and then crew runs getrealdeps.rb for us and adds the runtime dependencies it detects on the end. I have plans to deal with duplicate dependencies tree wide and alphabetise them, which this package will get picked up by anyway.

Reply:
The problem with crew running getrealdeps.rb is it doesn't check for duplicates.  @satmandu: Maybe it makes sense to document the `=> :build`, `#L` and `# R` nomenclature so we have a better understanding how the dependencies should be added to packages?  I'm still a bit foggy on how to clearly distinguish between everything.

Reply:
Should we move this to a github discussion? It's not really particularly relevant to this PR at this point.

Reply:
:woman_shrugging: this is the first time you've brought this up out of some 20 or so packages i've added this year, but ok

also, i'm working on a way to automate this, so no need to hold up any PRs over it in the future.

Reply:
Asking to sort dependencies is not new; especially with packages like this that have many dependencies.  We do this to prevent duplicates and with getrealdeps.rb there really isn't any reason why we cannot document packages correctly in a clean and consistent manner.